docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyONNXModelWithInMemoryImages.cs

```diff
@@ -26,8 +25,8 @@ public static void Example()
             // input /output of the used ONNX model.
             var dataPoints = new ImageDataPoint[]
             {
-                new ImageDataPoint(Color.Red),
-                new ImageDataPoint(Color.Green)
+                new ImageDataPoint(red: 255, green: 0, blue: 0), // Red color
+                new ImageDataPoint(red: 0, green: 128, blue: 0)  // Green color
```

Comment:
What's the reasoning behind this choice of green?

Reply:
`Color.Green` in `System.Drawing` is equal to [`#FF008000`](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.color.green?view=net-6.0). Green is `0x80` which is `128` in decimal.